doc/Internals.md

```diff
@@ -347,7 +347,7 @@ container.load(frontendLanguagesModule);
 
 ## Complete example
 
-If you wish to see the complete implementation of what I refered too in
+If you wish to see the complete implementation of what I referred too in
```

Comment:
I think for small typos like this we could push directly to master from the GH markdown editor (for those that can) so as to not clutter big PRs even more.

Reply:
Fixes that should be fixed on master in general  should be there asap to avoid duplicating work especially on long running PRs like this one